lib/web/command.go

```diff
@@ -228,10 +228,12 @@ func (h *Handler) executeCommand(
 
 		msgPayload, err := json.Marshal(struct {
```

Comment:
What consumes this json payload? Should we move the definition in some shared place?

Reply:
Moved.